src/features/offerv2/components/OfferContent/OfferContent.native.test.tsx

```diff
@@ -164,6 +172,14 @@ describe('<OfferContent />', () => {
     expect(screen.getByTestId('image-gradient')).toBeOnTheScreen()
   })
 
+  it('should display tag on offer image when enableOfferPreview feature flag activated', async () => {
+    renderOfferContent({})
+
+    await act(async () => {})
+
+    expect(screen.getByTestId('image-tag')).toBeOnTheScreen()
```

Comment:
```suggestion
    expect(await screen.findByTestId('image-tag')).toBeOnTheScreen()
```
ne marche pas ?

Reply:
ça marche ! C'est mieux ? J'essaye de l'utiliser partout où je peux dans le fichier de test ?

Reply:
Ouais, les `await act(async () => {})`, c'est à faire que dans le pire des cas, il vaut mieux un findBy ou un waitFor si possible.
Par contre, remplacer par un findBy, ne va pas marcher pour les .not.toBeOnTheScreen(), il va peut-être falloir garder le `await act(async () => {})` pour ces cas-là